### DISCUSSION TITLE
Re-run workflow actions after some time

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.Forms.Issues#1523] [Ideas]
Currently workflows run after submission or approval, which may fail. However it is possible to manually run these again under record.

I wonder if it would be possible to queue a job/task in case it fails, and e.g. try 5 or 10 times with increasing interval?
Furthermore if a record creation is cancelled via notification it is still possible to access submitted data, e.g. if one want to push data directly somewhere else, but don't need the record? In that case I guess the data would only be available on first run.

**Comment by TvGessel** [comment]
A possible workaround here is to build a custom workflow that includes its own retry logic for this specific scenario. There a no plans to include the request on the short term in the core Umbraco Forms packacge.
